### PR TITLE
Fixed code syntax issues

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

**Review by lyrixx — APPROVED** [review]

**Review by mleczakm — APPROVED** [review]

File: doctrine/resolve_target_entity.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ without making them hard dependencies.
 Background
 ----------
 
-Suppose you have an `InvoiceBundle` which provides invoicing functionality
-and a `CustomerBundle` that contains customer management tools. You want
+Suppose you have an ``InvoiceBundle`` which provides invoicing functionality
```
Comment (xabbuh) [inline doctrine/resolve_target_entity.rst]: IIRC we agreed on not enclosing bundle names with backticks at all.

File: templating/twig_extension.rst [hunk 2]
```diff
@@ -58,8 +58,8 @@ As an example you'll create a price filter to format a given number into price::
 
 .. tip::
 
-    Along with custom filters, you can also add custom `functions` and register
-    `global variables`.
+    Along with custom filters, you can also add custom functions and register
+    global variables.
```
Comment (xabbuh) [inline templating/twig_extension.rst]: Maybe these were supposed to link to other sections (note the entries at the end of the document)?

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Comment by xabbuh** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**